package.json

```diff
@@ -4,12 +4,14 @@
   "description": "IOpipe Lambda Analytics & Tracing Agent",
   "main": "dist/iopipe.js",
   "scripts": {
-    "build": "webpack --progress --profile --colors",
+    "build": "npm run webpack && npm run prettierDist",
     "eslint": "eslint src spec",
-    "jasmine": "istanbul cover jasmine --captureExceptions",
+    "jest": "jest",
     "prepare": "npm run build",
     "prepublishOnly": "npm run build",
-    "test": "npm run eslint && npm run build && npm run jasmine"
+    "prettierDist": "prettier dist/iopipe.js --single-quote --write",
```

Comment:
Won't running prettier on our dist add a bunch of whitespace, and thus increase size?